src/components/DeviceJob/steps.js

```diff
@@ -158,6 +159,55 @@ export const genuineCheck: Step = {
     ),
 };
 
+export const listApps: Step = {
+  Body: ({ meta }: *) =>
+    meta.genuineAskedOnDevice ? (
```

Comment:
wording a bit weird, => `allowManagerRequested` ?

Reply:
@gre I used that wording on my boilerplate code because it's what you added https://github.com/LedgerHQ/ledger-live-mobile/blame/c3124eeab0200b4283619113f380842ebe4c31c7/src/screens/PairDevices/PendingGenuineCheck.js#L17 🤷‍♂

Reply:
what i meant is it's actually not a genuine check, it's just a "allow manager" request that appear on device

Reply:
:ah: